CMakeLists.txt

```diff
@@ -349,8 +360,11 @@ if (NOT STB_IMAGE_FOUND)
         SHOW_PROGRESS)
     list(GET STB_IMAGE_WRITE_DOWNLOAD_STATUS_LIST 0
         STB_IMAGE_WRITE_DOWNLOAD_STATUS)
+    check_hash (http://mlpack.org/files/stb/hash.md5 "${CMAKE_BINARY_DIR}/deps/${STB_DIR}"
+        HASH_CHECK_FAIL)
```

Comment:
I added `http://www.mlpack.org/files/stb/hash.md5` so this should work now. :+1: